### DISCUSSION TITLE
Potential issue with useFormContext's submit function

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [airjp73/rvf#161] [Q&A]
When reading the useFormContext docs, I found this to not be true:
```
Submit
() => void
Submits the form, running all validations first.

Note: This is equivalent to clicking a button element with type="submit" or calling formElement.submit().
```

1. When calling the `submit` function returned from useFormContext, any clickHandler on the form's button with type="submit" are ignored.
2. When calling the `submit` function returned from useFormContext, the value for the form's button with type="submit" is not 
 submitted. To check this, give the button a name="test" and a value="123".

**Comment by airjp73** [comment]
I guess the wording here is a bit confusing. The `onSubmit` handler will be called as normal, but it doesn't stimulate any clicks. It's essential a wrapper around `formElement.requestSubmit()`.

PRs welcome with alternate wording! :)
